README.md

```diff
@@ -1,125 +1,55 @@
 # Shoulda Matchers [![Gem Version][version-badge]][rubygems] [![Build Status][travis-badge]][travis] ![Downloads][downloads-badge] [![Hound][hound-badge]][hound]
 
+![website][logo]
```

Comment:
This is a nitpick but I feel like the logo is super huge. Can we use an `img` tag here so we can specify a width here? I forgot if GitHub will let us do that or not.

Reply:
I made a PNG version of the logo here: https://matchers.shoulda.io/images/shoulda-matchers-logo.png. What do you think about using this instead?